src/store/products/actions/__test__/producst-actions.test.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ProductsActionTypes,
+  productsLoadStart,
+  productsLoadSuccess,
+  productsLoadError,
+ } from '..';
+import { Product } from '../../../../domain/Product';

```

Comment:
try using absolute routes: 'src/domain/Product', like this

Reply:
Done